### PR TITLE
filebrowser_s3 as its own module

### PR DESCRIPTION
**Pull request by Pomax**
there are debug prints etc that should be removed before we land!

### CODE REVIEW

File: app/networkapi/settings.py [hunk 1]
```diff
@@ -19,8 +19,8 @@
 # we rely on it being explicitly set (no default values) so that
 # we error out first.
 env = environ.Env(
-    DEBUG=(bool, False),
-    FILEBROWSER_DEBUG=(bool, False),
```
Comment (cadecairos) [inline app/networkapi/settings.py]: If we do commit these defaults, we need to verify we turn them off on staging/production.

File: docker-compose.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ services:
       - database
     volumes:
       - ./app:/app
-    command: bash -c "../prepare_container.sh && python /app/manage.py collectstatic --no-input && python /app/manage.py runserver 0:5000"
```
Comment (cadecairos) [inline docker-compose.yml]: lol.

File: Dockerfile [hunk 3]
```diff
@@ -3,6 +3,6 @@ FROM python:3.6
 WORKDIR /app
 EXPOSE 5000
 COPY ./requirements.txt .
-RUN pip install -r requirements.txt ipdb==0.9.3 --no-cache-dir --disable-pip-version-check
```
Comment (cadecairos) [inline Dockerfile]: should we really reinstall all packages each time, even if there are not version changes?

**Comment by cadecairos** [comment]
We need to remove the old code before we merge.

File: app/networkapi/settings.py [hunk 4]
```diff
@@ -42,7 +42,7 @@
 SECRET_KEY = env('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = FILEBROWSER_DEBUG = env('DEBUG')
```
Comment (cadecairos) [inline app/networkapi/settings.py]: This needs to be set to env('DEBUG')

File: docker-compose.yml (hunk 2 above)
Comment (Pomax) [inline docker-compose.yml]: it's weird, some of them take both `--no-input` and `--noinput`, and others *only* take `--noinput`, so when we added collectstatic `--no-input` did nothing =P

File: Dockerfile (hunk 3 above)
Comment (Pomax) [inline Dockerfile]: I guess if filebrowser_s3 is no longer subject to constant change, we don't need this change in the Dockerfile.

**Pull request merged**